README.md

```diff
@@ -2,29 +2,122 @@
 [![Codecov](https://codecov.io/gh/eclipse-iceoryx/iceoryx-rs/branch/master/graph/badge.svg?branch=master)](https://codecov.io/gh/eclipse-iceoryx/iceoryx-rs?branch=master)
 
 <p align="center">
 <img src="https://user-images.githubusercontent.com/8661268/114321508-64a6b000-9b1b-11eb-95ef-b84c91387cff.png" width="50%">
 </p>
 
-Experimental Rust bindings for the [iceoryx](https://github.com/eclipse-iceoryx/iceoryx) IPC middleware.
+Safe Rust bindings to [Eclipse iceoryx](https://github.com/eclipse-iceoryx/iceoryx).
 
-# clone and build
+1. [About](#about)
+2. [Examples](#examples)
+    - [How to start RouDi](#how-to-start-roudi)
+    - [Run the simple publisher and subscriber example](#run-the-simple-publisher-and-subscriber-example)
+3. [Limitations](#limitations)
 
-Clone the repository with
+## About
+
+Eclipse iceoryx is a true zero-copy, inter-process communication framework with the goal to boost
+autonomous driving with their demand on high data throughput and low latency. With this properties,
+iceoryx also fits well into other domains where low latency and transmitting large data structures
+is a concern. For a closer look about Eclipse iceoryx actually is, pleaes have a look at
+`Getting started` section on [iceoryx.io](https://iceoryx.io) or the
+[README.md](https://github.com/eclipse-iceoryx/iceoryx/blob/master/README.md) of the main project.
+
+The Rust bindings are a work in progress and and currently support only the pub-sub messaging pattern.
+Upcoming release will close the gap and the goal is to have the Rust bindings as first class citizen
+in the iceoryx ecosystem.
+
+This project started with the goal to create an nice looking introspection TUI in Rust and led to
+[iceray](https://crates.io/crates/iceray). Check it out.
+
+## Examples
+
+Before you can run the example you have to build them first with
 ```
-git clone https://github.com/elBoberido/iceoryx-rs.git
+cargo build --all --examples
 ```
 
-To build the examples run
+In order to run an iceoryx application, the `RouDi` deamon needs to run.
+
+### How to start RouDi
+
+`RouDi` is the cetral deamon which takes care of resource management and connects the services
+when they register. After the registration phase it is not involved in the communication anymore.
+
+You can find more information [here](https://iceoryx.io/v2.0.2/getting-started/overview/#roudi)
+and if you are more of a visual person, just scroll up a little bit on that page to view a nice animation.
+
+In case you have iceoryx installed on your system, you can use the `iox-roudi` binary from that installation.
+If that's not the case, you can run `RouDi` with the following command from the root of your crate.
+
 ```
-cargo build --all --examples
+find target -type f -wholename "*/iceoryx-install/bin/iox-roudi" -exec {} \;
+```
+
+### Run the simple publisher and subscriber example
+
+The most simple examples are the `publisher_simple` and `subscriber_simple`.
```

Comment:
A good introductory example is ...

In the end plural is here maybe the wrong grammar since it is one example but two binaries.

Reply:
I went with `A good introductory example to demonstrate the inter-process communication are ...`